src/libostree/libostree-released.sym

```diff
@@ -551,6 +551,7 @@ LIBOSTREE_2019.2 {
 
 LIBOSTREE_2019.3 {
 global:
+  ostree_async_progress_copy_state;
```

Comment:
Should this not be in a new section of the symbol file?

It also needs to be added to the tests: see the Jenkins failure:
```
Verifying all public symbols are documented:
--- expected-documented.txt	2019-11-04 21:29:16.961623193 +0000
+++ found-documented.txt	2019-11-04 21:29:16.969623259 +0000
@@ -1,4 +1,3 @@
-ostree_async_progress_copy_state
 ostree_async_progress_finish
 ostree_async_progress_get
 ostree_async_progress_get_status
ERROR: tests/test-symbols.sh - too few tests run (expected 3, got 1)
ERROR: tests/test-symbols.sh - exited with status 1
```

Reply:
I think if we add the API without rebasing it would need to go in here, since 2019.3 is the current version on EOS. Upstream it would probably need to go in `libostree-devel.sym` and then get copied into a new section here.